### ISSUE TITLE
When considering keyboard focus order, try to minimize bouncing around the screen for pan/zoom.

### ISSUE THREAD

**Issue by samreid** [phetsims/projectile-data-lab#64]

**Comment by samreid** [comment]
I implemented the request in #60, not sure if there is more to do for this issue. @matthew-blackman can this be closed?

**Comment by samreid** [comment]
There is some bouncing around, but the keyboard order is as designed in https://github.com/phetsims/projectile-data-lab/issues/44.  @matthew-blackman can this be closed?

**Comment by matthew-blackman** [comment]
Let's get some feedback from @terracoda before next steps. @terracoda, can you review the keyboard navigation order and weigh in on the latest updates?

**Comment by terracoda** [comment]
And I am also wondering if the Timing Control Node can just stay closer to the data navigator thing (the thing on top of the filed sign). Maybe like where it is positioned on screen 4. Could it stay near there? It doesn't seem related to the check boxes.

**Comment by matthew-blackman** [comment]
@terracoda sounds good! We implemented the updates to the keyboard focus order to have less bouncing around. Ready for your review.

@jessegreenberg and @terracoda - @samreid and I noticed that we were only using the pdomControlAreaNode, and were wondering if we should be splitting things up between that and that pdomPlayAreaNode. It did seem more flexible to put everything in a single parent since we can easily change the order, but we may not be considering all needs here. Can you weigh in?

**Comment by terracoda** [comment]
Great question about pdomPlayAreaNode  and pdomControlAreaNode.

For long term accessibility and consistency across all sims, you should be using both.

You want your essential interactions to be in the pdomPlayAreaNode, and anything secondary or tertiary in the pdomControlAreaNode.

Those decision are not always easy to make, especially when there are many controls, like there are in this sim, but you want to think about what interactions are truly essential for the main learning goals:

1. The launcher and the field are essential
2. The histogram and the launcher projectile confirmations are likely pretty essential.
3. The tape measure, timer and view tools are very likely best in the Control Area.

Everything in the Play Are comes before the Control Area in the Tab order.

**Comment by jessegreenberg** [comment]
+1 for what @terracoda said. Assigning contents to the different parents can categorize page content nicely under "Play Area" and "Control Area" headings in the Parallel DOM for a screen reader user. Using both will have no impact on alt input (other than that play area contents come before the control area contents).

**Comment by samreid** [comment]
So what parts of this simulation would be in the "play area"?

**Comment by jessegreenberg** [comment]
The things that are most important to the pedagogy/interaction should go in the play area. Sometimes it has been everything except the "Reset All" button. 

Here is how some other sims have divvied it up:
https://github.com/phetsims/gravity-force-lab/blob/7ae7430c16eac840d2c2d24de62772bec466bc8f/js/view/GravityForceLabScreenView.js#L264-L276

https://github.com/phetsims/ratio-and-proportion/blob/b20713b13f371ba5771cedfb21cbac39fcf13c9e/js/common/view/RAPScreenView.ts#L463-L472

https://github.com/phetsims/greenhouse-effect/blob/47967c6759a1f691004ce17c0cf9a1c9f322b357/js/common/view/GreenhouseEffectScreenView.ts#L142-L151

**Comment by samreid** [comment]
OK I opened side issue #107. This issue can remain about "When considering keyboard focus order, try to minimize bouncing around the screen for pan/zoom. The status is that @matthew-blackman and @samreid implemented @terracoda recommendations, and the keyboard order is ready for @terracoda review.  We will preserve the keyboard order in https://github.com/phetsims/projectile-data-lab/issues/107

**Comment by matthew-blackman** [comment]
The keyboard focus order has been updated to balance usability and avoid bouncing around during pan/zoom. Here is the order we are currently using:

1. Launch button
2. Launch mode
3. Launch panel (to set up an experiment)
4. Field selector (to navigate between experiments)
5. Eraser
6. Histogram
7. Global tools
8. Projectile selector
9. Reset all

This keyboard order is working well for usability and to minimize bouncing. Let's continue working on separating things into a playArea and controlArea in https://github.com/phetsims/projectile-data-lab/issues/107. Closing.
